RCTMapboxGL.ios.js

```diff
@@ -4,6 +4,9 @@ var React = require('react-native');
 var { NativeModules, requireNativeComponent } = React;
 
 var MapMixins = {
+  _findHandle(mapRef) {
+    return React.findNodeHandle(this.refs[mapRef]);
+  },
   setDirectionAnimated(mapRef, heading) {
```

Comment:
@christocracy why was this helper function added? Can't this be accomplished in the same manner as the rest as mixin functions? IE 

```
NativeModules.MapboxGLManager.addAnnotations(React.findNodeHandle(this.refs[mapRef]), annotations);
```